Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/day/tabs/common/DiaryCircumstancesTextView.kt

```diff
@@ -0,0 +1,58 @@
+package de.rki.coronawarnapp.contactdiary.ui.day.tabs.common
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.EditText
+import android.widget.ImageView
+import androidx.constraintlayout.widget.ConstraintLayout
+import de.rki.coronawarnapp.R
+import timber.log.Timber
+
+class DiaryCircumstancesTextView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : ConstraintLayout(context, attrs, defStyleAttr) {
+
+    private val input: EditText
+    private val infoButton: ImageView
+
+    private var afterTextChangedListener: ((String) -> Unit)? = null
+
+    init {
+        LayoutInflater.from(context).inflate(R.layout.view_diary_circumstances_textview, this, true)
+
+        input = findViewById<EditText>(R.id.input).apply {
+            setOnFocusChangeListener { v, hasFocus ->
+                if (hasFocus) {
+                    Timber.v("Focused on %s", v)
+                } else {
+                    Timber.v("Lost focus on %s", v)
```

Comment:
On my device the focus isn't lost when putting the app in background (and then close it) 🤔

Reply:
Will offer a solution to this in a follow up PR.